perllib/FixMyStreet/Cobrand/Bexley/Garden.pm

```diff
@@ -404,4 +409,12 @@ sub waste_get_paye_narrative {
     return "Garden Waste Service Payment - Reference: $id Contract: $uprn";
 }
 
+=item * Anyone can modify garden subs XXX Is that right?
+
+XXX TODO can't modify in renewal period
```

Comment:
I think this is handled at the start of `sub garden_modify` in Waste.pm controller, which doesn't proceed if renewal is due:

https://github.com/mysociety/fixmystreet/blob/b1bf5844591b4e07340fb69589ff6fe76d3d73ef/perllib/FixMyStreet/App/Controller/Waste.pm#L1269

Reply:
Yeah, I added a test for this as well, forgot these comments were hanging around!